packages/odo-expandable/src/expandable.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * @fileoverview An basic, expandable component that has both a trigger
+ * and a target to open.
+ *
+ * @author Matt Zaso
+ */
+import Settings from './settings';
+import ExpandableGroup from './expandable-group';
```

Comment:
We have a circular dependency here.
```
Circular dependency: packages/odo-expandable/src/expandable.js -> packages/odo-expandable/src/expandable-group.js -> packages/odo-expandable/src/expandable.js
```

Maybe `expandable.js` should be renamed to `expandable-item.js` and this file changes to only export `initializeAll`, `ExpandableGroup`, and ExpandableItem`. Thoughts?

Reply:
I think this is the best solution right now based on some of the initializers that I want to add here in addition. I'll update this. Thanks!